### PR TITLE
Nit fixes on documentation

### PR DESCRIPTION
**Pull request by tusharxoxoxo**
### Description
It mainly includes typos, sentence constructions, and punctuations. Fixes are reported by Grammarly.

#### All Submissions:

* [x] I've signed all my commits
* [x] I followed the [contribution guidelines](https://github.com/bitcoindevkit/bdk/blob/master/CONTRIBUTING.md)
* [x] I ran `cargo fmt` and `cargo clippy` before committing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,20 +20,20 @@ The below directories (a separate repository in the case of bdk-swift) include i
...
-2. Update mapping `bdk.udl` file with new `dictionary`
+#### For pass-by-value objects
+1. Create a new rust struct with only fields that are supported by UniFFI types
+2. Update mapping `bdk.udl` file with a new `dictionary`
 
-#### For pass by reference values
+#### For pass-by-reference values
 1. Create wrapper rust struct/impl with only fields that are `Sync + Send`
-2. Update mapping `bdk.udl` file with new `interface`
+2. Update mapping `bdk.udl` file with new a `interface`
```
Comment (thunderbiscuit) [inline README.md:36]: This one doesn't make sense.

File: README.md [hunk 2]
```diff
@@ -10,7 +10,7 @@
 The workspace in this repository creates the `libbdkffi` multi-language library for the Rust-based 
 [bdk] library from the [Bitcoin Dev Kit] project.
 
-Each supported language and the platform(s) it's packaged for has its own directory. The Rust code in this project is in the bdk-ffi directory and is a wrapper around the [bdk] library to expose its APIs in a uniform way using the [mozilla/uniffi-rs] bindings generator for each supported target language.
+Each supported language and the platform(s) it's packaged for have their own directories. The Rust code in this project is in the bdk-ffi directory. It is a wrapper around the [bdk] library to expose its APIs uniformly using the [mozilla/uniffi-rs] bindings generator for each supported target language.
```
Comment (thunderbiscuit) [inline README.md:13]: I'm ok with the `uniformly` change. The rest is better as is IMO.

**Review by thunderbiscuit — CHANGES_REQUESTED** [review]
Thanks for your contribution!

Good little fixes in there, but not all of them are good IMO. Let's remove a few and this will be ready to go.

**Comment by tusharxoxoxo** [comment]
@thunderbiscuit I have made the changes as per your suggestions.

**Comment by thunderbiscuit** [comment]
Perfect. Can you squash those two commits into one? After that I'll give the ACK.

**Comment by tusharxoxoxo** [comment]
@thunderbiscuit I have squashed it

**Comment by thunderbiscuit** [comment]
Yes it just means approving the PR.

Can you tell us more about who you are and why you decided to contribute to the repos? I see you made similar grammarly small fixes to bdk.

**Review by thunderbiscuit — APPROVED** [review]
ACK 38c078b70fbc634af60282d1f20d551e07664280

**Comment by thunderbiscuit** [comment]
I can't rebase and push for you in this case because you're using the master branch on your fork for this PR.

Can you rebase to pick up the latest changes on master and force push?

**Comment by notmandatory** [comment]
@tusharxoxoxo, as with your other PRs in the `bdk` and `bdk-cli` repo I'm closing this PR because the changes don't constitute a meaningful improvement. Feel free to open a new PR if you'd like to make any more substantial additions or fixes.

**Comment by tusharxoxoxo** [comment]
Okay sure

**Pull request closed without merging**